src/main/java/seedu/contax/ui/TextStyleHelper.java

```diff
@@ -0,0 +1,106 @@
+package seedu.contax.ui;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontPosture;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.Text;
+
+public class TextStyleHelper {
+
+    /**
+     * Parser to identify bold/italic components within a string
+     * @param text String to parse
+     * @return List of Javafx Text elements
+     */
+    public static List<Text> formattedTextParser(String text) {
```

Comment:
I think it might be rather problematic if the markdown `**` is not opened and closed correctly. Also, I think monospace will override all bold and italic formatting.

Reply:
yea i thought of that too, but I don't really have an alternative solution at the moment. Since this is a function that we mostly use on our own when presenting text, I don't think it'll be a huge problem as long as we do our own due diligence in ensuring any text that needs this styling has the proper syntax. 

if anyone has any suggestions on the logic of how we can improve this pls do suggest.